components/pkg-export-container/defaults/Dockerfile.hbs

```diff
@@ -42,7 +42,12 @@ RUN \
     # communication pipe.
     mkdir /root && chmod 750 root && \
     mkdir /tmp && chmod 1777 /tmp && \
-    mkdir -p /var/tmp && chmod 1777 /var/tmp
+    mkdir -p /var/tmp && chmod 1777 /var/tmp && \
+
+    # This is currently needed in order for non-root containers to
+    # function... this ends up being their home directory, and we
+    # expect to find key cache directory there right now.
+    mkdir -p /.hab/cache/keys && chmod -R 770 /.hab
```

Comment:
Double checking my understanding here:  This path is the result of `hab` being run as a user, so we use the `$HOME/.hab` directory for a cache, and `$HOME` being unset in the running container,  correct?

Reply:
yup